src/setup/joystick.c

```diff
@@ -1021,9 +1028,25 @@ void ConfigJoystick(void)
                    TXT_TABLE_OVERFLOW_RIGHT,
                    TXT_TABLE_EMPTY,
                    TXT_TABLE_EMPTY,
+                   NULL);
 
-                   TXT_NewSeparator("Buttons"),
+    if (gamemission >= heretic) // heretic, hexen, strife
```

Comment:
expecting some resistance to this (but I like it)

Reply:
What if it is instead `if (gamemission != doom)`?

Reply:
I prefer `gamemission == heretic || gamemission == hexen || gamemission == strife`.

A good way to think about it is to ask: if another game was added, what would happen?

Reply:
I've gone with @fragglet suggestion. I'm just a sucker for trying to squeeze more utility out of enums...